Code/Framework/AzFramework/AzFramework/Input/Channels/InputChannelDigitalWithSharedModifierKeyStates.h

```diff
@@ -53,14 +53,13 @@ namespace AzFramework
         //! \return True if the modifier key is active, false otherwise
         bool IsActive(ModifierKeyMask modifierKey) const;
 
-    private:
         ////////////////////////////////////////////////////////////////////////////////////////////
         //! Set the active state of the specified modifier key
         //! \param[in] modifierKey The modifier key to set
         //! \param[in] active The active state to set
         void SetActive(ModifierKeyMask modifierKey, bool active);
-        friend class InputChannelDigitalWithSharedModifierKeyStates;
 
+    private:
```

Comment:
The previous class was impossible test test so I decided to make this interface public - the chances of misuse are pretty minor and now it's easy to mock `ModifierKeyStates` in a unit/integration test